modules/cdp-engine/src/lib.rs

```diff
@@ -119,7 +120,12 @@ pub mod module {
 	use super::*;
 
 	#[pallet::config]
-	pub trait Config: frame_system::Config + loans::Config + SendTransactionTypes<Call<Self>> {
+	pub trait Config:
+		frame_system::Config
+		+ loans::Config
+		+ pallet_timestamp::Config<Moment = Moment>
```

Comment:
Better to use this UnixTime

https://github.com/paritytech/substrate/blob/4898eb350d636d439827cb43f0f26e72e66492fa/frame/support/src/traits/misc.rs#L160